keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -38,6 +38,7 @@ import org.springframework.stereotype.Component
 
 @Component
 class ActuationPauser(
+  // TODO: replace use of individual repos with KeelRepository
```

Comment:
If this only cares about resources, I don't see the need to replace it with the `KeelRepository`